### PR TITLE
pr split six - add five files

### PR DESCRIPTION
**Pull request by michaelkernaghan**
Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: integration-tests/contract-no-annotations-call-by-index-methodsObject.spec.ts [hunk 1]
```diff
@@ -3,13 +3,15 @@ import { noAnnotCode, noAnnotInit } from "./data/token_without_annotation";
...
+  describe(`Test contract origination with no annotations calling methods by index using methodObjects through contract api: ${rpc}`, () => {
+    /** The goal of the test is to verify a contract entrypoint call using the methodsObject method (contract.methodsObject) 
+     *  in case of a contract having no annotation in its code. */
+    
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    it('Test contract with no annotations for methods', async (done) => {
+    it('Verify contract.originate for a contract with no annotations for methods using methodObjects', async (done) => {
```
Comment (roxaneletourneau) [inline integration-tests/contract-no-annotations-call-by-index-methodsObject.spec.ts:14]: Suggestion: `Verify contract.originate for a contract without annotation and call entry points using methodObjects`

**Review by roxaneletourneau — CHANGES_REQUESTED** [review]

File: integration-tests/contract-manager-scenario.spec.ts [hunk 2]
```diff
@@ -5,12 +5,12 @@ import { MANAGER_LAMBDA, Protocols, protocols } from "@taquito/taquito";
...
   const Tezos = lib;
 
-  describe(`Manager TZ: ${rpc}`, () => {
+  describe(`Test TZ Manager through contract api: ${rpc}`, () => {
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    it('test manager transfers scenarios', async (done) => {
+    it('Verify contract.transfer from one Baker to another', async (done) => {
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/contract-manager-scenario.spec.ts:13]: changed as suggested

File: integration-tests/contract-manager-that-calls-failwith-contract.spec.ts [hunk 3]
```diff
@@ -6,13 +6,13 @@ import { MANAGER_LAMBDA } from "@taquito/taquito";
...
 
-  describe(`Test contract that calls 2nd contract that FAILs: ${rpc}`, () => {
+  describe(`Test contract origination of a contract that calls 2nd contract that FAILs through contract api: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    test('Catches a Failwith from a contract called via a manager contract',  async (done) => {
+    test('Verify that the method call (called via a manager contract) for a contract with code FAILWITH that links to another contract will fail',  async (done: () => void) => {
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/contract-manager-that-calls-failwith-contract.spec.ts:15]: changed as suggested, and void removed

File: integration-tests/contract-multisig-transfer-implicit.spec.ts [hunk 4]
```diff
@@ -5,12 +5,12 @@ import { genericMultisig } from "./data/multisig";
 CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
   const Tezos = lib;
 
-  describe(`Generic Multisig: ${rpc}`, () => {
+  describe(`Test originating multisig contract through contract api: ${rpc}`, () => {
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/contract-multisig-transfer-implicit.spec.ts:8]: changed as suggested

File: integration-tests/contract-multisig-transfer-implicit.spec.ts [hunk 5]
```diff
@@ -5,12 +5,12 @@ import { genericMultisig } from "./data/multisig";
...
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    test('test manager transfers scenarios', async (done) => {
+    test('Verify contract.originate for a contract with multiple signatures', async (done: () => void) => {
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/contract-multisig-transfer-implicit.spec.ts:13]: changed as suggested and void removed

File: integration-tests/contract-no-annotations-call-by-index-methodsObject.spec.ts (hunk 1 above)
Comment (michaelkernaghan) [reply integration-tests/contract-no-annotations-call-by-index-methodsObject.spec.ts:14]: changed as suggested

Comment (roxaneletourneau) [reply integration-tests/contract-no-annotations-call-by-index-methodsObject.spec.ts:14]: @michaelkernaghan looks like the change was not pushed

**Pull request merged**